docs/current_docs/quickstart/snippets/test/php/src/HelloDagger.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Attribute\Doc;
+use Dagger\Container;
+use Dagger\Directory;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class HelloDagger
+{
+    #[DaggerFunction]
+    #[Doc('Return the result of running unit tests')]
+    public function test(Directory $source): string
+    {
+        return $this
+            // get the build environment container
+            // by calling another Dagger Function
+            ->buildEnv($source)
+            // call the test runner
+            ->withExec(['npm', 'run', 'test:unit', 'run'])
```

Comment:
why are we running javascript unit tests in a PHP project? this makes no sense

```
./vendor/bin/phpunit
```

is correct

Reply:
Same as https://github.com/dagger/dagger/pull/9509#discussion_r1946909601